### DISCUSSION TITLE
What is the difference between inventor hub and prime hub?

### DISCUSSION THREAD

**Discussion by myhewen** [pybricks/support#1579] [Q&A] [answered]
What is the difference between inventor hub and prime hub? Looks like everything is the same except for the color

**Comment by dlech** [comment]
And the USB product ID and the bootloader version. That's about it really.

**Comment by ggramlich** [comment, marked as answer]
@myhewen Just to mention that @dlech 's answer is true only for the use with pybricks.
If you try to use the Lego software, you will find that the newest version of the Spike app cannot be used with the inventor hub. The older Spike app allowed flashing its firmware on the prime hub.
